libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewListener.kt

```diff
@@ -129,4 +121,19 @@ abstract class NavigationViewListener {
      * Called when the info panel behavior updates to collapsed.
      */
     open fun onInfoPanelCollapsed() = Unit
+
+    /**
+     * Called when new instance of [MapView] is created.
+     *
+     * @param mapView the new [MapView] instance
+     */
+    open fun onMapViewAttached(mapView: MapView?) = Unit
```

Comment:
```suggestion
    open fun onMapViewAttached(mapView: MapView) = Unit
```

No need to handle null maps attached here. It will just not be called when there is not a map